dxcint/dependencies/config/node.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "node",
```

Comment:
nodejs?

Reply:
right, that's for `nodejs` but when you unpack the binary, it's located in `tarball/bin/node` - this is to avoid too much of hardcoding and just use a dependency name. Would be essential to consider how that naming works if we add new dependencies as tarball dependency.